### PR TITLE
Projects page header

### PR DESCRIPTION
**Pull request by vikdoro**
https://trello.com/c/G2WaL4E6

### CODE REVIEW

File: src/elements/kw-projects-header/kw-projects-header.html [hunk 1]
```diff
@@ -270,52 +318,50 @@ <h5 class="staff-pick-memo">Remember! Share your creation for the chance to be S
...
+                    type: Object,
+                    value: 
+                        {
+                            speakerIntro: ["speaker_hello", "speaker_cannot_think", "speaker_code_me_loop",
+                                    "speaker_sound", "speaker_sound_loop", "speaker_button_trigger", "speaker_drum_machine"],
+                            pixelIntro: ["pixel_lights_on", "pixel_color_picker", "pixel_color_random_loop",
+                                    "pixel_point", "pixel_emoji", "pixel_colorful_point_loop"],
+                            screenIntro: ["screen_draw_pixel", "screen_pixel_flood", "screen_colorful_pixels",
+                                "screen_colorful_squares", "screen_colorful_circles", "screen_colorful_circles_loop"]
+                        }
```
Comment (abridger) [inline src/elements/kw-projects-header/kw-projects-header.html]: Could we format the arrays a little more clearly, down the page?

File: src/elements/kw-projects-header/kw-projects-header.html [hunk 2]
```diff
@@ -233,29 +225,85 @@
...
+                    <div class="project-group">
+                        <div class="project-group-header">
+                            <iron-image src="/assets/projects/simple-progress/hello.svg" preload fade sizing="contain"></iron-image>
+                            <div class="project-group-info">
+                                <h4>The power to make your computer speak</h4>
+                            </div>
+                        </div>
+                        <kano-carousel>
+                            <button type="button" slot="left-button" class="left-button" carousel-control-left>&lt;</button>
+                                <div id="speaker-puzzles" class="step-display" slot="content">
```
Comment (abridger) [inline src/elements/kw-projects-header/kw-projects-header.html]: `speaker-puzzles` should be `speaker_puzzles`

File: src/elements/kw-projects-header/kw-projects-header.html [hunk 3]
```diff
@@ -233,29 +225,85 @@
...
+                    <div class="project-group">
+                        <div class="project-group-header">
+                            <iron-image src="/assets/projects/simple-progress/screen.svg" class="screen" preload fade sizing="contain"></iron-image>
+                            <div class="project-group-info">
+                                <h4>The power to draw</h4>
+                            </div>
+                        </div>
+                        <kano-carousel>
+                            <button type="button" slot="left-button" class="left-button" carousel-control-left>&lt;</button>
+                                <div id="screen-puzzles" class="step-display" slot="content">
```
Comment (abridger) [inline src/elements/kw-projects-header/kw-projects-header.html]: Assuming `screen-puzzles` should be `screen_puzzles`

File: src/elements/kw-projects-header/kw-projects-header.html [hunk 4]
```diff
@@ -270,52 +318,50 @@ <h5 class="staff-pick-memo">Remember! Share your creation for the chance to be S
...
+                        }
+                    }) 
+                }
             },
-            _checkboxChanged (e) {
-                let target = e.path ? e.path[0] : e.target;
-                if (target.checked) {
-                    this.fire('expand');
+            _calculateLabel (index, array) {
+                //TODO something else instead of passing an empty string to avoid alignment issues
```
Comment (abridger) [inline src/elements/kw-projects-header/kw-projects-header.html]: Could you just implement fallback content in the slot instead?

File: src/elements/kw-projects-header/kw-projects-header.html [hunk 5]
```diff
@@ -40,93 +47,122 @@
...
+            .step-icon.current {
+                background-image: url("/assets/projects/simple-progress/make-button.svg");
+                background-size: 82%;
+                width: 110px;
+                height: 60px;
+            }
+            /*TODO refactor*/
+            .step-icon.current a,
+            .step-icon.done a,
+            kano-step:first-child .step-icon:not(.done) a{
```
Comment (abridger) [inline src/elements/kw-projects-header/kw-projects-header.html]: space between `a` and `{` please

File: src/elements/kw-projects-header/kw-projects-header.html [hunk 6]
```diff
@@ -40,93 +47,122 @@
...
+                @apply --layout-horizontal;
+                @apply --layout-center;
+                margin: auto 10px;
             }
-            input[type=checkbox]:checked ~ .header .toggle-container {
-                transform: translateY(0);
-                opacity: 1;
-                margin-top: -10px;
+            .project-group-info {
+                @apply(--layout-vertical);
```
Comment (abridger) [inline src/elements/kw-projects-header/kw-projects-header.html]: Inconsistency with other `@apply` methods, which aren't bracketed

File: src/elements/kw-app/kw-app.html [hunk 7]
```diff
@@ -263,9 +263,7 @@
                 });
             },
             _computeNavbar (view) {
-                if (view === 'start') {
-                    this.simplePage = true;
-                }
+                this.simplePage = view === 'start'
```
Comment (abridger) [inline src/elements/kw-app/kw-app.html:266]: Nice

**Comment by vikdoro** [comment]
thanks guys! I got rid of the dom repeat

**Comment by vikdoro** [comment]
@abridger i changed the bits after your review. a few of those (computeLabel and slug array) have gone completely as everything is hard-coded now for speed

**Review by abridger — APPROVED** [review]
Looks good

**Pull request merged**